### DISCUSSION TITLE
FullStack .Net

### DISCUSSION THREAD

**Discussion by rogercomp** [balta-io/forum#667] [Tira Dúvidas] [answered]
No video [Endpoint para recuperar o pedido pelo número] o verbo utilizado no GetOrderByNumberEndpoint  foi o MapPost, o correto nao seria o MapGet?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rogercomp, sim é GET, acredito que nas próximas aulas eu já corrijo!
